airflow/jobs/scheduler_job.py

```diff
@@ -1050,23 +754,17 @@ class SchedulerJob(BaseJob):  # pylint: disable=too-many-instance-attributes
 
     def __init__(
             self,
-            dag_id: Optional[str] = None,
-            dag_ids: Optional[List[str]] = None,
             subdir: str = settings.DAGS_FOLDER,
             num_runs: int = conf.getint('scheduler', 'num_runs'),
+            num_times_parse_dags: int = -1,
```

Comment:
Is this somehow configurable or we are just hardcoding it?

Reply:
It's currently used in tests to limit things. Used, but I chose not to expose it to users.

Reply:
If we are using it only for tests (I suppose to set it to something other than -1) then wouldn't it be better to use something like `if unittest then...` as we already do in many places?

Reply:
It's not used in every test is the issue, so we don't want to always set it when unittest is enabled.

Reply:
Got it, no strong opinion here but please, add a comment around this line to leave trace to why we do this

Reply:
Will do

Reply:
Added in d09c846d9